File: app/src/main/java/org/schulcloud/mobile/data/model/responseBodies/GeogebraResponse.java
package org.schulcloud.mobile.data.model.responseBodies;

/**
 * Date: 2/18/2018
 */
public class GeogebraResponse {
    public Responses responses;

    public static class Responses {
        public Response response;

        public static class Response {
            public Item item;

            public static class Item {
                public String previewUrl;
            }
        }
    }
}

File: app/src/main/java/org/schulcloud/mobile/models/content/GeogebraResponse.kt
package org.schulcloud.mobile.models.content

class GeogebraResponse {
    var responses: Responses? = null

    class Responses {
        var response: Response? = null

        class Response {
            var item: Item? = null

            class Item {
                var previewUrl: String? = null
            }
        }
    }
}
